### PR TITLE
Update doc for ShareWithTeam cmdlets

### PR DESCRIPTION
**Pull request by zhongxlmicrosoft**
Adds doc for new ShareWithTeam cmdlets doc and update existing cmdlets doc

Adding doc for new cmdlets
- Get-SharedWithTeam
- Get-TeamAllChannel
- Get-TeamIncomingChannel
- Get-SharedWithTeamuser
- Remove-SharedWithTeam
- Get-AssociatedTeam

Updating doc for the existing cmdlets
- Add-TeamChannelUser supporting xtenant user scenario.
- Get-TeamChannel description.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: teams/teams-ps/teams/Get-AssociatedTeam.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+### Example 1
+```PowerShell
+Get-AssociatedTeam
+```
+
+Returns associated teams of current user.
+
+### Example 2
+```PowerShell
+Get-TeamChannel -user example@example.com
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-AssociatedTeam.md:35]:
```suggestion
Get-AssociatedTeam -user example@example.com
```

File: teams/teams-ps/teams/Get-AssociatedTeam.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+```
+
+Returns associated teams of given user email.
+
+### Example 3
+```PowerShell
+Get-TeamChannel -user 0e4249a7-6cfd-8c93-a510-91cda44c8c73
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-AssociatedTeam.md:42]:
```suggestion
Get-AssociatedTeam -user 0e4249a7-6cfd-8c93-a510-91cda44c8c73
```

**Review by get-itips — CHANGES_REQUESTED** [review]
Please add missing common parameters, inputs, outputs, notes section in the new articles that you are submitting, thanks

File: teams/teams-ps/teams/Remove-SharedWithTeam.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+## SYNOPSIS
+This cmdlet supports usharing a channel with a team.
+
+## SYNTAX
+```PowerShell
+Remove-SharedWithTeam -HostTeamId <String> -ChannelId <String> -SharedWithTeamId <String>
+```
+
+## DESCRIPTION
+This cmdlet supports usharing a channel with a team.
```
Comment (get-itips) [inline teams/teams-ps/teams/Remove-SharedWithTeam.md:22]: Please fix the description

**Review by get-itips — CHANGES_REQUESTED** [review]
Please see my comments, thanks

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+## DESCRIPTION
+This cmdlet supports retrieving teams with which specified channel is shared.
+
+## EXAMPLES
+
+### Example 1
+```PowerShell
+Get-AssociatedTeam -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:28]:
```suggestion
Get-Get-SharedWithTeam -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
```

File: teams/teams-ps/teams/Get-SharedWithTeamUser.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+```
+
+## DESCRIPTION
+This cmdlet supports retrieving users of a shared with team.
+
+## EXAMPLES
+
+### Example 1
+```PowerShell
+Get-AssociatedTeamUser -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
```
Comment (get-itips) [inline teams/teams-ps/teams/Get-SharedWithTeamUser.md:28]:
```suggestion
Get-SharedWithTeamUser -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
```

File: teams/teams-ps/teams/Get-AssociatedTeam.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+This cmdlet supports retrieving all associated teams of a user, including teams which host shared channels.
+
+## EXAMPLES
+
+### Example 1
+```PowerShell
+Get-AssociatedTeam
+```
+
+Returns associated teams of current user.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-AssociatedTeam.md:31]:
```suggestion
Returns associated teams of the current user.
```

File: teams/teams-ps/teams/Get-AssociatedTeam.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+### Example 2
+```PowerShell
+Get-TeamChannel -user example@example.com
+```
+
+Returns associated teams of given user email.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-AssociatedTeam.md:38]:
```suggestion
Returns associated teams of the given user email.
```

File: teams/teams-ps/teams/Get-AssociatedTeam.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+### Example 3
+```PowerShell
+Get-TeamChannel -user 0e4249a7-6cfd-8c93-a510-91cda44c8c73
+```
+
+Returns associated teams of given user id.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-AssociatedTeam.md:45]:
```suggestion
Returns associated teams of the given user ID.
```

File: teams/teams-ps/teams/Get-AssociatedTeam.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+## PARAMETERS
+
+### -User
+User's UPN (user principal name - e.g.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-AssociatedTeam.md:50]:
```suggestion
User's UPN (user principal name, for example
```

File: teams/teams-ps/teams/Get-AssociatedTeam.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+johndoe@example.com) or user id (e.g. 0e4249a7-6cfd-8c93-a510-91cda44c8c73)
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-AssociatedTeam.md:51]:
```suggestion
johndoe@example.com) or user ID (for example 0e4249a7-6cfd-8c93-a510-91cda44c8c73).
```

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+schema: 2.0.0
+author: zhongxl
+ms.author: zhongxl
+ms.reviewer: dedaniel, robharad
+---
+
+# Get-SharedWithTeam
+
+## SYNOPSIS
+This cmdlet supports retrieving teams with which specified channel is shared.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:14]:
```suggestion
This cmdlet supports retrieving teams with which a specified channel is shared.
```

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+
+## SYNTAX
+```PowerShell
+Get-SharedWithTeam -HostTeamId <String> -ChannelId <String> [-SharedWithTeamId <String>]
+```
+
+## DESCRIPTION
+This cmdlet supports retrieving teams with which specified channel is shared.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:22]:
```suggestion
This cmdlet supports retrieving teams with which a specified channel is shared.
```

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+
+## EXAMPLES
+
+### Example 1
+```PowerShell
+Get-AssociatedTeam -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
+```
+
+Returns teams with which specified channel is shared.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:31]:
```suggestion
Returns teams with which a specified channel is shared.
```

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+
+### Example 2
+```PowerShell
+Get-AssociatedTeam -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
+-SharedWithTeam d2aad370-c6ca-438b-b4d7-05f0aa911a7b
+```
+
+Returns detail of a team with which specified channel is shared.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:39]:
```suggestion
Returns detail of a team with which a specified channel is shared.
```

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+
+## PARAMETERS
+
+### -HostTeamId
+Team id of the host team (group id)
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:44]:
```suggestion
Team ID of the host team (group ID)
```

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ChannelId
+Thread id of the shared channel
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:59]:
```suggestion
Thread ID of the shared channel.
```

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 17]
```diff
@@ -0,0 +1,86 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -SharedWithTeamId
+Team id of the shared with team
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:74]:
```suggestion
Team ID of the shared with team.
```

File: teams/teams-ps/teams/Get-SharedWithTeamUser.md [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+-SharedWithTeam d2aad370-c6ca-438b-b4d7-05f0aa911a7b
+```
+
+Returns users of a team with which specified channel is shared.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeamUser.md:32]:
```suggestion
Returns users of a team with which a specified channel is shared.
```

File: teams/teams-ps/teams/Get-SharedWithTeamUser.md [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+
+### Example 2
+```PowerShell
+Get-AssociatedTeam -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
+-SharedWithTeam d2aad370-c6ca-438b-b4d7-05f0aa911a7b -Role owner
+```
+
+Returns owners of a team with which specified channel is shared.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeamUser.md:40]:
```suggestion
Returns owners of a team with which a specified channel is shared.
```

File: teams/teams-ps/teams/Get-SharedWithTeamUser.md [hunk 20]
```diff
@@ -0,0 +1,102 @@
...
+
+## PARAMETERS
+
+### -HostTeamId
+Team id of the host team (group id)
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeamUser.md:45]:
```suggestion
Team ID of the host team (group ID)
```

File: teams/teams-ps/teams/Get-SharedWithTeamUser.md [hunk 21]
```diff
@@ -0,0 +1,102 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ChannelId
+Thread id of the shared channel
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeamUser.md:60]:
```suggestion
Thread ID of the shared channel.
```

File: teams/teams-ps/teams/Get-SharedWithTeamUser.md [hunk 22]
```diff
@@ -0,0 +1,102 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -SharedWithTeamId
+Team id of the shared with team
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeamUser.md:75]:
```suggestion
Team ID of the shared with team.
```

File: teams/teams-ps/teams/Get-SharedWithTeamUser.md [hunk 23]
```diff
@@ -0,0 +1,102 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -Role
+Filter the results to only users with the given role: Owner or Member.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeamUser.md:90]:
```suggestion
Filters the results to only users with the given role of "Owner" or "Member".
```

File: teams/teams-ps/teams/Get-TeamAllChannel.md [hunk 24]
```diff
@@ -0,0 +1,70 @@
...
+```PowerShell
+Get-TeamAllChannel -GroupId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -MembershipType Shared
+```
+
+Returns all shared channels of a team.
+
+## PARAMETERS
+
+### -GroupId
+Group id of the team
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-TeamAllChannel.md:43]:
```suggestion
Returns the Group ID of the team.
```

File: teams/teams-ps/teams/Get-TeamAllChannel.md [hunk 25]
```diff
@@ -0,0 +1,70 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -MembershipType
+Membership type of the channel to display, Standard, Private, or Shared
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-TeamAllChannel.md:58]:
```suggestion
Membership type of the channel to display; Standard, Private, or Shared
```

File: teams/teams-ps/teams/Get-TeamChannel.md [hunk 26]
```diff
@@ -8,11 +8,9 @@ schema: 2.0.0
 # Get-TeamChannel  
 
 ## SYNOPSIS
-
-Get all the channels for a team.
+Get the channels hosted by a team.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-TeamChannel.md:11]:
```suggestion
This cmdlet gets the channels hosted by a team.
```

File: teams/teams-ps/teams/Get-TeamIncomingChannel.md [hunk 27]
```diff
@@ -0,0 +1,48 @@
...
+```PowerShell
+Get-TeamIncomingChannel -GroupId 126b90a5-e65a-4fef-98e3-d9b49f4acf12
+```
+
+Returns incoming channels of a team.
+
+## PARAMETERS
+
+### -GroupId
+Group id of the team
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-TeamIncomingChannel.md:36]:
```suggestion
Group ID of the team
```

File: teams/teams-ps/teams/Remove-SharedWithTeam.md [hunk 28]
```diff
@@ -0,0 +1,79 @@
...
+schema: 2.0.0
+author: zhongxl
+ms.author: zhongxl
+ms.reviewer: dedaniel, robharad
+---
+
+# Remove-SharedWithTeam
+
+## SYNOPSIS
+This cmdlet supports usharing a channel with a team.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-SharedWithTeam.md:14]:
```suggestion
This cmdlet supports sharing a channel with a team.
```

File: teams/teams-ps/teams/Remove-SharedWithTeam.md (hunk 3 above)
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-SharedWithTeam.md:22]:
```suggestion
This cmdlet supports sharing a channel with a team.
```

File: teams/teams-ps/teams/Remove-SharedWithTeam.md [hunk 29]
```diff
@@ -0,0 +1,79 @@
...
+
+## EXAMPLES
+
+### Example 1
+```PowerShell
+Remove-SharedWithTeam -HostTeamId 126b90a5-e65a-4fef-98e3-d9b49f4acf12 -ChannelId 19:cUfyYYw3h_t-1KG8-WkvVa7KLEsIx-JHmyeG43VJojg1@thread.tacv2
+-SharedWithTeam d2aad370-c6ca-438b-b4d7-05f0aa911a7b
+```
+
+Ushares a channel with a team.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-SharedWithTeam.md:32]:
```suggestion
Shares a channel with a team.
```

File: teams/teams-ps/teams/Remove-SharedWithTeam.md [hunk 30]
```diff
@@ -0,0 +1,79 @@
...
+
+## PARAMETERS
+
+### -HostTeamId
+Team id of the host team (group id)
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-SharedWithTeam.md:37]:
```suggestion
Team ID of the host team (Group ID).
```

File: teams/teams-ps/teams/Remove-SharedWithTeam.md [hunk 31]
```diff
@@ -0,0 +1,79 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -ChannelId
+Thread id of the shared channel
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-SharedWithTeam.md:52]:
```suggestion
Thread ID of the shared channel.
```

File: teams/teams-ps/teams/Remove-SharedWithTeam.md [hunk 32]
```diff
@@ -0,0 +1,79 @@
...
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -SharedWithTeamId
+Team id of the shared with team
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-SharedWithTeam.md:67]:
```suggestion
Team ID of the shared with team.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @zhongxl1993, kindly commit the suggested changes. Many thanks.

File: teams/teams-ps/teams/Add-TeamChannelUser.md [hunk 33]
```diff
@@ -104,12 +111,28 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -TenantId
```
Comment (dariomws) [inline teams/teams-ps/teams/Add-TeamChannelUser.md:114]: You must add this parameter to the Syntax section.

File: teams/teams-ps/teams/Get-TeamIncomingChannel.md [hunk 34]
```diff
@@ -0,0 +1,48 @@
...
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
```
Comment (dariomws) [inline teams/teams-ps/teams/Get-TeamIncomingChannel.md:48]: You are missing common parameters, output, etc

**Review by dariomws — CHANGES_REQUESTED** [review]

**Comment by yogkumgit** [comment]
@zhongxl1993 Please implement the suggested changes. Thanks

File: teams/teams-ps/teams/Get-AssociatedTeam.md [hunk 35]
```diff
@@ -0,0 +1,81 @@
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-AssociatedTeam.md:67]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Get-SharedWithTeam.md [hunk 36]
```diff
@@ -0,0 +1,104 @@
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeam.md:90]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Get-SharedWithTeamUser.md [hunk 37]
```diff
@@ -0,0 +1,120 @@
...
+Position: Named
+Default value: None
+Accept pipeline input: True
+Accept wildcard characters: False
+```
+
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-SharedWithTeamUser.md:107]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Get-TeamAllChannel.md [hunk 38]
```diff
@@ -0,0 +1,88 @@
...
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-TeamAllChannel.md:74]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Get-TeamIncomingChannel.md [hunk 39]
```diff
@@ -0,0 +1,66 @@
...
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-TeamIncomingChannel.md:52]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

File: teams/teams-ps/teams/Remove-SharedWithTeam.md [hunk 40]
```diff
@@ -0,0 +1,94 @@
...
+Required: True
+Position: Named
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### CommonParameters
+This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable.
+For more information, see about_CommonParameters (https://go.microsoft.com/fwlink/?LinkID=113216).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-SharedWithTeam.md:83]:
```suggestion
This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable, -InformationAction, -InformationVariable, -OutVariable, -OutBuffer, -PipelineVariable, -Verbose, -WarningAction, and -WarningVariable. For more information, see [about_CommonParameters](https://go.microsoft.com/fwlink/?LinkID=113216).
```

**Comment by yogkumgit** [comment]
@zhongxlmicrosoft Please implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@zhongxlmicrosoft Gentle reminder.

**Comment by yogkumgit** [comment]
@zhongxlmicrosoft Please implement the suggested changes so that merge can be initiated. In the event of no response we may proceed to close this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Comment by tiburd** [comment]
@zhongxlmicrosoft 

Hello,

At your earliest convenience, please complete your account setup so that your GitHub and Microsoft accounts are connected:
- [Set up your accounts](https://review.docs.microsoft.com/en-us/office-authoring-guide/contribute-get-started-setup-github-ro?branch=main)

Thanks!

**Pull request merged**